src/core/n-control.c

```diff
@@ -254,6 +254,36 @@ enum {
 	return R_ARG1;
 }
 
+/***********************************************************************
+**
+*/	REBNATIVE(protectedq)
```

Comment:
what does the `q` at the end mean? or is it just a typo?

Reply:
`q` means `question mark` as it is native definition for Rebol's `protected?` function. It's how it was defined in [original sources](https://github.com/rebol/rebol/blob/25033f897b2bd466068d7663563cd3ff64740b94/src/tools/make-boot.r#L136-L149).